src/rulesets/Base/base.jl

```diff
@@ -75,6 +75,32 @@ function rrule(::typeof(hypot), z::Complex)
     return (Ω, hypot_pullback)
 end
 
+# `sin`
+
+function rrule(::typeof(sin), x::Number)
+    sinx, cosx = sincos(x)
+    sin_pullback(dy) = (NO_FIELDS, cosx * dy)
+    return (sinx, sin_pullback)
+end
+
+function frule((_, dy), ::typeof(sin), x::Number)
+    sinx, cosx = sincos(x)
+    return (sinx, cosx)
```

Comment:
Perhaps this is why the test is failing.
```suggestion
    return (sinx, cosx * Δx)
```